docs/content/docs/testing/devtools.md

```diff
@@ -0,0 +1,71 @@
+
+Once you have completed the above steps, you can simply launch your Fluid application in a browser window and start the browser DevTools by right-clicking and selecting `Inspect`. In the browser DevTools window, you should see a tab for `Fluid Developer Tools`
+
+## Container Information
+
+DevTool shows all the containers in your application, their current state (connected, attached, disconnected) and allows developers to modify the state (disconnect, close). Developers can also see a lot of container state changes with timestamps.
+
+![A screenshot showing container information](/images/container_info.png)
+
+-   **Disconnect/Reconnect Container** button disconnects it from the Fluid service
+-   **Close Container** button detaches from the Fluid service
+
+![A screenshot showing container state changes](/images/container_states.png)
+
+These buttons can be used to test the following scenarios:
+
+-   **Offline state** - Test how your app will behave if connection drops in the middle of a collab session
+-   **Merge edge cases** - Test how your app reacts to various merging scenarios when multiple users work offline and come back online
+-   **Container cleanup** - Test how your app behaves when a container is closed and disposed off.
+
+## Data Visualization
+
+DevTool shows the current container data including all the DDSs that are part of the container. Currently, the DevTool supports renderers for `SharedTree`, `SharedMap`, `SharedString`, `SharedCounter` and `Sequences`. Because of the extensible model of the framework, you can also build and add renderers for custom DDSs.
+
+![A screenshot showing container data visualization](/images/container_data_viz.png)
+
+The DDS visualizer updates in real-time as local and remote changes impact the container data. Getting a peek at the underlying data and checking it against what's rendered in your app should allow you to look for any inconsistencies between the data and your app UI.
+
+## Audience Info
+
+DevTool displays a list of users that are connected to the container(s) in the application and their permission level (read/write). The tool also shows a log of users joining and leaving the container. This can be used to debug user presence indicators in your apps.
+
+![A screenshot showing container audience information](/images/container_audience_states.png)
+
+## Framework Event Logs
+
+DevTool outputs a log of events happening in the framework including info, performance (e.g. Ops latency) and errors.
+These logs provide a peek at the inner workings of Fluid that you can use to monitor your apps
+
+![A screenshot showing container event logs](/images/container_events.png)
+
+## Ops Latency Telemetry Graph
+
+DevTool shows a graph of network requests and a breakdown of their processing times. Developer can see the network request time broken down into:
```

Comment:
A bit tricky because multiple ops could have been sent in a single network request, so I'd suggest something like this. If "operations" at this point is assumed to be understood by the reader as Fluid ops, the other addition to that sentence could be removed.
```suggestion
DevTool shows a graph of the operations it generates to handle real-time collaboration and a breakdown of their processing times. Developer can see that time broken down into:
```